src/Network/HttpClient.php

```diff
@@ -102,26 +83,14 @@ private function getClient()
      * Sends request to Google Analytics.
      *
      * @param string $url
-     * @param SingleParameter[] $singleParameters
-     * @param CompoundParameterCollection[] $compoundParameters
      * @param boolean $nonBlocking
      * @return AnalyticsResponse
      */
-    public function post($url, array $singleParameters, array $compoundParameters, $nonBlocking = false)
+    public function post($url, $nonBlocking = false)
```

Comment:
Test cases failed? You mean in your fork? Because tests are green here...

I didn't know about the `@internal` tag, I see there is also an `@api` tag, I'll proceed to include those in the code, thanks.